src/gleam/float.gleam

```diff
@@ -403,7 +403,10 @@ fn do_product(numbers: List(Float), initial: Float) -> Float {
 }
 
 /// Generates a random float between the given minimum and maximum values.
-///
+/// if `inclusive < exclusive`, returns a value in `[inclusive, exclusive)`
+/// if `exclusive < inclusive`, returns a value in `(exclusive, inclusive]`
```

Comment:
Could you write this in plain English please? I think most people won't know what `[` and `)` mean. I had to look it up.